pyproject.toml

```
@@ -59,7 +59,7 @@ dev-mkdocs = [
   "mkdocs-macros-plugin == 1.0.5",
   "mkdocs-material == 9.5.25",
   "mkdocstrings[python] == 0.25.1",
-  "frequenz-repo-config[lib] == 0.9.2",
+  "mkdocstrings-python == 1.9.2",  "frequenz-repo-config[lib] == 0.9.2",
```

```suggestion
  "mkdocstrings-python == 1.9.2",
  "frequenz-repo-config[lib] == 0.10.0",
```

And you should search for the other instances of `frequenz-repo-config` and update it to `0.10.0` (there are 3 or 4 in total in this file).